chord_metadata_service/mohpackets/apis/ingestion.py

```diff
@@ -94,11 +90,11 @@ def create_biomarkers(
     return create_instances(payload, Biomarker)
 
 
-@router.post("/chemotherapies/")
-def create_chemotherapies(
-    request, payload: List[ChemotherapyIngestSchema], response: HttpResponse
+@router.post("/systemictherapies/")
+def create_systemictherapies(
```

Comment:
If a similar function is `create_primary_diagnoses`, this should follow that convention.
```suggestion
def create_systemic_therapies(
```